src/MetadataUtility/Metadata/SupportFiles/SupportFile.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="SupportFile.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group.
+// </copyright>
+
+namespace MetadataUtility.Metadata.SupportFiles
+{
+    public abstract class SupportFile
+    {
+        public string File { get; set; }
```

Comment:
is this the file contents or the file path?

Reply:
File path, updated the name here.